homeassistant/components/edl21/sensor.py

```diff
@@ -72,6 +72,15 @@
         icon="mdi:flash",
     ),
     # C=1: Active power +
+    # D=7: Current value
+    # E=0: Total
+    SensorEntityDescription(
+        key="1-0:1.7.0*255",
+        name="Positive active instantaneous power",
```

Comment:
Can you please make this new sensor translatable by adding `translation_key` instead of `name`.
More details can under [entity name translations](be found https://developers.home-assistant.io/blog/2023/03/27/entity_name_translations/)